packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -0,0 +1,434 @@
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/form-pill-group"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/form-pill-group/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Form Pill Group"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/form-pill-group"
+  storybookUrl="/?path=/story/components-form-pill-group--basic"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+## Guidelines
+
+### About Form Pill Group
+
+<Paragraph>
+  {props.pageContext.frontmatter.description} They are used almost exclusively in multi-select editing situations.
+</Paragraph>
+
+A Form Pill Group can be used on it's own to represent selection across a number of fields, such as displaying currently applied filters when filtering data across a number of different fields. Alternatively, a Form Pill Group can be paired directly to a single input field, such as a Combobox, to represent multiple selection.
+
+#### Accessibility
+
+The Form Pill Group provides a number of accessibility features. The only requirement from you, is to provide the Pill Group a descriptive label via the `aria-label` React prop.
+
+##### Keyboard navigation
+
+The Form Pill Group is focusable, but only one pill is focusable at a time. This means the Pill Group is a single tab stop to a keyboard user. Once focused within the Pill Group the following keyboard interactions apply.
+
+- Left and Right arrow keys move focus within the group
+- If selectable, spacebar and enter will toggle pill selection
+- If a pill has a supplied action via `onClick`, enter key will trigger that action
+- If dismissable, a Pill can be removed by pressing the delete or backspace key
+
+<Box marginBottom="space70">
+  <AspectRatio ratio="16:9">
+    <iframe
+      src="https://www.loom.com/embed/1bea986ec6a24cd494950e2c12f10562"
+      frameborder="0"
+      webkitallowfullscreen
+      mozallowfullscreen
+      allowfullscreen
+      title="Silent demo on keyboard navigation"
+    ></iframe>
+  </AspectRatio>
+</Box>
```

Comment:
I love this video!! Succinct but detailed, and I feel like I got a lot out of it even just watching it the first time! It pairs quite nicely with the list above there. Big win here!

Reply:
🔥